docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -562,6 +565,65 @@ The default value is `fail`, which causes the migrate procedure to throw an
 exception if subdirectories are found. Set the value to `true` to migrate
 nested directories, or `false` to ignore them.
 
+(iceberg-add-files)=
+#### Add files
+
+The connector can add files from tables or locations if
+`iceberg.add-files-from-location-procedure.enabled` is set to `true` for the catalog.
+
+Use the procedure `system.add_files_from_table` to add existing files from the Hive 
```

Comment:
> Use the procedure `system.add_files_from_table` to add existing files from the Hive  table

Why not then `system.add_files_from_hive_table`?

Reply:
Let's add a failure test importing files from a `tpch` catalog table.
Let's add as well a test importing files from a `iceberg` catalog table (Iceberg tables, compared to tpch tables contain files as well).

Reply:
> Let's add a failure test importing files from a tpch catalog table.

I don't think we can add such a test because this procedure doesn't allow a catalog argument.

> Let's add as well a test importing files from a iceberg catalog table (Iceberg tables, compared to tpch tables contain files as well).

We already have `TestIcebergAddFilesProcedure.testAddFilesFromIcebergTable` that verifies `Adding files from non-Hive tables is unsupported` error.